include/zephyr/net/net_pkt.h

```diff
@@ -244,6 +244,8 @@ struct net_pkt {
 	uint8_t tx_timestamping : 1; /** Timestamp transmitted packet */
 	uint8_t rx_timestamping : 1; /** Timestamp received packet */
 #endif
+	uint8_t loopback: 1; /* Set to 1 if this packet is a loopback packet */
```

Comment:
nit, to align with other entries
```suggestion
	uint8_t loopback : 1; /* Set to 1 if this packet is a loopback packet */
```

Reply:
Plus we should also cover the new flag in `net_pkt_clone()`.